runtime/src/utils.rs

```diff
@@ -0,0 +1,157 @@
+use crate::{
+    Balance, BloatBondFeePerByte, Call, ExistentialDeposit, Runtime, StorageDepositCleanupProfit,
+    UncheckedExtrinsic,
+};
+use codec::{Encode, FullCodec, MaxEncodedLen};
+use frame_support::{
+    storage::{
+        generator::{StorageDoubleMap, StorageMap},
+        StoragePrefixedMap,
+    },
+    traits::StorageInfoTrait,
+    weights::{GetDispatchInfo, WeightToFee},
+    StorageHasher,
+};
+pub use sp_runtime::Perbill;
+use sp_std::mem::size_of;
+
+/// Compute total fee for executing a call
+pub fn compute_fee(call: Call) -> Balance {
```

Comment:
So initially I was thinking about using `TransactionPayment::compute_fee` approach as described in https://github.com/Joystream/joystream/issues/4300, but it had a few major drawbacks:
- The main issue was that it was reading a storage value (fee multiplier) which meant the result would change depending on it. This was very risky, because the runtime constants shouldn't change.
- It didn't work for genesis config values, only for the pallet's `Config` trait parameters

So I implemented the logic of `TransactionPayment::compute_fee` in this function while disregarding the fee multiplier (so assuming it's `1`). The formula is quite simple as it's just:
```
inclusion_fee = base_fee + length_fee + [targeted_fee_adjustment * weight_fee];
```
_Source: https://docs.rs/pallet-transaction-payment/3.0.0/pallet_transaction_payment/_

So if we disregard  `targeted_fee_adjustment` (fee multiplier), it's just:
```
inclusion_fee = base_fee + length_fee + weight_fee;
```